gitlab/resource_gitlab_group_share_group_test.go

```diff
@@ -46,8 +60,13 @@ func TestAccGitlabGroupShareGroup_import(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				// create shared groups
-				Config: testAccGitlabGroupShareGroupConfig(randName, "guest", "2099-03-03"),
-				Check:  testAccCheckGitlabGroupSharedWithGroup(randName, "2099-03-03", gitlab.GuestPermissions),
```

Comment:
Looks like this `Check` is just removed, nothing added back, was that intentional or just needs to be added back?

Reply:
I noticed other `import` tests skipped this `check` so I wasn't sure it was needed.  I went ahead and added it back. 

Thanks!